novawallet/Modules/AssetsSearch/AssetSearch/AssetsSearchPresenter.swift

```diff
@@ -29,40 +31,118 @@ class AssetsSearchPresenter: AssetsSearchPresenterProtocol {
         self.localizationManager = localizationManager
     }
 
+    func selectGroup(with symbol: AssetModel.Symbol) {
+        processGroupSelectionWithCheck(
+            symbol,
+            onSingleInstance: { chainAsset in
+                processAssetSelected(with: chainAsset.chainAssetId)
+            },
+            onMultipleInstances: { _ in }
```

Comment:
Why we are not handling group selection in any way? I think at least we should expand or close the group, so the same logic as in the main list

Reply:
The group is expanded/collapsed by the Collection Manager. Since we don't need to do anything else in expandable list, we ignore it.